recipes/openmc-plotter/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "openmc-plotter" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmc-dev/plotter/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f54644486eed1b87ae4959a464bbce0a55f625508ac99afd2b5606c7312811e8
+
+build:
+  entry_points:
+    - openmc-plotter=openmc_plotter.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

Comment:
Is the only reason you are skipping it a missing dependency? In that case it can be noarch so that users can install this package on windows, once the missing dependency becomes available.

Reply:
@paulromano can you weigh in on this? If I don't skip Windows it tries to install OpenMC which is not available through conda-forge, which leads to a failure. This OpenMC plotter package is from the same org as OpenMC that causes this issue.

Reply:
If it fails because of the missing dependency it doesn't matter. Then it can be noarch.

Ref: Third note here: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

Reply:
This is the [issue](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=514492&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b&l=808) that occurs if I add `noarch: python` instead of `skip: true  # [win]`. What's the correct way to handle this error?

Reply:
The CI isn't loading properly for me so I can't see it, but if it's just the windows build failing because the dependency is not available, _it's fine for the build to fail in this PR for that reason_ - the actual noarch package from the feedstock is done on linux only, so it'll pass the build - it just won't be installable on windows, unless the upstream package becomes available (e.g. IF OpenMC were to add a windows build, you would presumably want this package to just work without changing it).

Reply:
@ndevenish Is Azure also not loading old logs for you? I’ve been seeing that a couple if times now and wasn’t sure if it’s something on my side.

Reply:
Yes, it was just windows failing in CI for that reason, so I will let that go then.